xbmc/network/upnp/UPnP.cpp

```diff
@@ -249,7 +249,10 @@ class CMediaBrowser : public PLT_SyncMediaBrowser,
             if (time < 0) time = 0;
             curr_value.Append(NPT_String::Format("<upnp:lastPlaybackPosition>%ld</upnp:lastPlaybackPosition>",
                                                  (long)item.GetVideoInfoTag()->m_resumePoint.timeInSeconds));
+            curr_value.Append(NPT_String::Format("<upnp:lastPlayerState>%s</upnp:lastPlayerState>",
```

Comment:
Is `upnp:lastPlayerState` defined in the UPnP specs? If not you need to use another XML namespace like `xbmc:` (which we use in otherplaces).